static/app/components/sidebar/index.tsx

```diff
@@ -176,16 +196,18 @@ function Sidebar() {
     return () => bcl.remove('collapsed');
   }, [collapsed]);
 
-  const hasPanel = !!activePanel;
-  const hasOrganization = !!organization;
-  const orientation: SidebarOrientation = horizontal ? 'top' : 'left';
+  // Add sidebar hasNewNav classname to body
+  useEffect(() => {
+    const bcl = document.body.classList;
 
-  const sidebarItemProps = {
-    orientation,
-    collapsed,
-    hasPanel,
-    organization,
-  };
+    if (hasNewNav) {
+      bcl.add('hasNewNav');
+    } else {
+      bcl.remove('hasNewNav');
+    }
+
+    return () => bcl.remove('hasNewNav');
+  }, [hasNewNav]);
```

Comment:
thoughts on using a data property for this rather than classlist? The data property can also be used in css so your selector could check to see if a parent has the data property set.

Reply:
🤷 

since we're already utilizing this property pattern, i'd rather keep it to a single pattern than try to introduce a new one.

This instance in particular is a really odd one.
This is actually for a global style to update the margins of the body - which we cannot set on the actual global body tag until we can actually pull the flag from the org state...